dali/python/nvidia/dali/external_source.py

```diff
@@ -14,9 +14,8 @@
 
 # custom wrappers around ops
 from nvidia.dali import backend as _b
-import inspect
-import functools
 import nvidia.dali.types
+from nvidia.dali._utils.external_source_impl import get_callback_from_source, accepted_arg_count
```

Comment:
Now these functions are public in `external_source` module. I'm okayish with that, but was that intended?